packages/bedrock-components/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@saasquatch/bedrock-components",
+  "version": "1.0.0",
+  "description": "Bedrock Components for SaaSquatch.",
+  "main": "dist/index.cjs.js",
+  "module": "dist/custom-elements/index.js",
+  "es2015": "dist/esm/index.mjs",
+  "es2017": "dist/esm/index.mjs",
+  "types": "dist/custom-elements/index.d.ts",
+  "collection": "dist/collection/collection-manifest.json",
+  "collection:main": "dist/collection/index.js",
+  "unpkg": "dist/bedrock-components/bedrock-components.esm.js",
+  "files": [
+    "dist/",
+    "loader/",
+    "docs/"
+  ],
+  "license": "MIT",
+  "author": "SaaSquatch Team",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/saasquatch/program-tools.git",
+    "directory": "packages/bedrock-components"
+  },
+  "scripts": {
+    "build": "stencil build --docs",
+    "start": "stencil build --dev --watch --serve",
+    "test": "stencil test --spec --e2e",
+    "test.watch": "stencil test --spec --e2e --watchAll",
+    "generate": "stencil generate"
+  },
+  "dependencies": {
+    "@saasquatch/component-boilerplate": "^1.0.0",
+    "@saasquatch/stencil-grapes-plugin": "^1.0.0",
```

This should be a dev dependency

```suggestion
    "@saasquatch/stencil-grapes-plugin": "^1.0.0",
```